api-docs/swagger.yaml

```diff
@@ -92,6 +92,43 @@ paths:
           description: Application error
           schema:
             $ref: '#/definitions/ErrorResponse'
+  '/{domain}/{envId}/{appGuid}/data/collections/upload':
+    parameters:
+      - $ref: "#/parameters/domain"
+      - $ref: "#/parameters/envId"
+      - $ref: "#/parameters/appGuid"
+    post:
+      summary: Upload collection(s)
+      description: |
+        Upload json, bson, or csv file to import collection(s)
+      operationId: uploadCollection
```

Comment:
I misunderstood this when you asked. Can you change this to `insertCollection` please?